pipelines/rj_crm__disparo_template/utils/validators.py

```
@@ -180,6 +182,123 @@ def log_validation_summary(stats: ValidationStats, context: str = ""):
+        campaign_name se encontrado na tabela, ou None se não encontrado.
+    """
+    from time import sleep  # pylint: disable=C0415
+
+    from basedosdados import Base  # pylint: disable=E0611, E0401, C0415
+    from google.cloud import bigquery  # pylint: disable=E0611, E0401, C0415
+
+    query = f"""
+        SELECT COUNT(*) AS total
+        FROM `rj-crm-registry.brutos_salesforce.jornada`
+        WHERE hsm.nome_hsm = '{campaign_name}'
+    """
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)